src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -754,9 +767,11 @@ function getVisibilityMessage(buttonElem) {
     	url: url,
     	data: formData,
     	success: function(data) {
-    		$(form).find('.visibilityMessage').html(formatVisibilityMessageHtml(data.visibilityMessage));
-    		$(form).find('.visibilityOptions').hide();
-    		$(form).find('.visibilityMessage').show();
+    	    var $formVisibility = $(form).find('.visibilityMessage');
+    	    
+    	    $formVisibility.html(formatVisibilityMessageHtml(data.visibilityMessage));
+    	    $formVisibility.hide();
+    	    $formVisibility.show();
```

Comment:
This seems incorrect.

The original code was 
<code> 
$(form).find('.visibilityOptions').hide();
$(form).find('.visibilityMessage').show();
</code>